tower_cli/conf.py

```diff
@@ -219,16 +229,18 @@ def __getattr__(self, key):
             except configparser.NoOptionError:
                 continue
 
-            # We have a value; it may or may not be a string, though, so
-            # try to return it as an int, float, or boolean (in that order)
-            # before falling back to the string value.
-            type_method = ('getint', 'getfloat', 'getboolean')
-            for tm in type_method:
-                try:
-                    value = getattr(parser, tm)('general', key)
-                    break
-                except ValueError:
-                    pass
+            # We have a value; try to get its type and return it accordingly
+            try:
+                if CONFIG_PARAM_TYPE[key] == (click.STRING or click.Choice):
```

Comment:
I expect that this won't work for the case of `click.Choice`

```python
>>> 2 == (1 or 2)
False
```

Reason is because

```python
>>> (1 or 2)
1
```

So you have to put the `or` between equality comparisons, like `2 == 1 or 2 == 2`, which would return True.

Alternatively, you can do `2 in (1, 2)`

Reply:
@AlanCoding Yup. Bad use of logical operators from me :(. Corrected that.